Docs grammar and typos check (#6064)
vuestorefront/vue-storefront: packages/core/docs/shopify/use-search.md

# useSearch composable

> `useSearch` composition API function is responsible, as its name suggests, for interactions with searching from your eCommerce. This function returns the following values:

Reviewer: ```suggestion
`useSearch` composition API function allows searching for products. This function returns the following values:
```
